### PR TITLE
Bugfix missing sample files

### PR DESCRIPTION
**Pull request by jacobvjk**
After erroneously deleting the sample_files directory in a past clean up, this PR adds it again.
This folder contains the files required to set up a new PACTA_analysis project.

What happens when a new project is initialised via the project_settings.yml, is that the copy_files() function https://github.com/2DegreesInvesting/PACTA_analysis/blob/fd928ba2e520d91806ba93004203070ed004d2f1/0_global_functions.R#L184
will add some basic folder structure and parameter files in the target directory. These files are taken from the sample_files folder, that I removed by accident.

It's easy to validate by setting the project name in the project_settings.yml to something random such as `test_12345` and twodii_internal to `TRUE` and running the first script. This creates the corresponding directory on the 2dii dropbox.

@jdhoffa @maurolepore

### CODE REVIEW

File: sample_files/10_folder_structures/start_folders/00_Log_Files/dummy.txt
Comment (maurolepore) [inline sample_files/10_folder_structures/start_folders/00_Log_Files/dummy.txt:1]: Great use of a comment.

File: sample_files/20_input_files/ReportParameters.yml
Comment (maurolepore) [inline sample_files/20_input_files/ReportParameters.yml:33]: Nit: The style seems inconsistent. Best is to stick to one (say `snake_case`). A popular style guide is https://style.tidyverse.org/, which includes naming conventions.

**Comment by maurolepore** [comment]
@jacobvjk, 
I prefer to not merge this myself because I lack context or tests. And you seem to lack admin privileges to merge this PR.
But let me know if this is an emergency and we can have someone to approve me giving you the required permission.

File: sample_files/20_input_files/ReportParameters.yml
Comment (jacobvjk) [reply sample_files/20_input_files/ReportParameters.yml:33]: It most definitely is inconsistent, but I feel uncomfortable changing variable names that propagate through this repo. Which is a good sign this whole thing ought to get a re-write at some point.

**Comment by jacobvjk** [comment]
@maurolepore I of course validated with the example I described above. But it's fine not to merge today. I will try to get @jdhoffa to do it tomorrow. It's urgent enough not to wait until Clare is back (in two weeks), but not urgent enough to warrant escalating it to emergency level today :)

**Review by jdhoffa — APPROVED** [review]
Lgtm

**Comment by jacobvjk** [comment]
@jdhoffa if you find time ti merge this today, that would be much appreciated. I don't have write access.

**Comment by maurolepore** [comment]
Do we need to give @jacobvjk admin permission?
It seems useful and safe -- given he understand git well.

cc' @jdhoffa, @2diiKlaus, @Clare2D

**Comment by 2diiKlaus** [comment]
Sounds reasonable to me. Especially in times when @Clare2D is on vacation.

**Comment by jdhoffa** [comment]
agreed, I think @jacobvjk should have admin permissions

**Comment by maurolepore** [comment]
Thanks! 

@jacobvjk, I increased your access permission. You now have rights to "write" to this repo. This is the minimum level I believe would give you the power you need. This can be changed at https://github.com/2DegreesInvesting/PACTA_analysis/settings/access.

image

**Comment by Clare2D** [comment]
This works for me too!

**Pull request merged**
